core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -111,21 +110,21 @@ object Kleisli extends KleisliInstances with KleisliFunctions with KleisliExplic
     }
 }
 
-sealed private[data] trait KleisliFunctions {
+private[data] sealed trait KleisliFunctions {
```

Comment:
Was this done for a particular reason? :)

Reply:
no, just a rebase issue - I'll revert